app/jobs/cleanup_account_job.rb

```diff
@@ -1,18 +1,32 @@
 class CleanupAccountJob < ActiveJob::Base
   def perform(account)
-    cleanup_solr(account)
-    cleanup_fedora(account)
+    # Ensure all account endpoints are active/configured
+    account.switch do
+      cleanup_solr(account)
+      cleanup_fedora(account)
+      # Redis doesn't need 'account', as switch changes Redis::Namespace to correct namespace
+      cleanup_redis
```

Comment:
NOTE: I've decided to keep the `account.switch` around *all* cleanup tasks. Deletions are inherently dangerous. The `account.switch` ensures the proper solr/fedora connections are enabled and redis namespace is configured. While the `switch` may not be required for most solr/fedora cleanup (as only one solr and one fedora is used by default in multitenancy setup), IMHO this is much safer than the assumption that one solr and one fedora will always be used.

Love to hear others thoughts on this, however.